presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -315,9 +323,15 @@ export class PresentationManager {
   constructor(props?: PresentationManagerProps) {
     this._props = props ?? {};
     this._detail = new PresentationManagerDetail(this._props);
+    this.activeLocale = this._props.defaultLocale; // eslint-disable-line deprecation/deprecation
+
+      this._localizationHelper = new LocalizationHelper({ getLocalizedString: props.getLocalizedString ?? getLocalizedStringEN });
   }
 
-  /** Get / set active locale used for localizing presentation data */
+  /**
+   *Get / set active locale used for localizing presentation data
+   * @deprecated
+   */
   public get activeLocale(): string | undefined { return this._detail.activeLocale; }
```

Comment:
I think this can be moved from `_detail` to `PresentationManager` until it can be removed entirely.